codespell_lib/data/dictionary.txt

```diff
@@ -13915,6 +13965,7 @@ recommneds->recommends
 recommpile->recompile
 recommpiled->recompiled
 recompence->recompense
+recomplie->recompile
```

Comment:
Suggest recomply which appears to be valid despite Firefox not liking it: https://www.merriam-webster.com/dictionary/recomply

Reply:
Done. I made:
recomplie->recompile, recomply,

it's common on software to re-compile software and so on.